cmd/helpers.go

```diff
@@ -53,3 +56,24 @@ func flagStringNullValueOrNil(flags *pflag.FlagSet, flag string) (*null.String,
 	// if not defined, return nil
 	return nil, nil
 }
+
+// buildVarsToMap is used to convert incoming build variable arguments into a structure consumable by the graphQL call
+func buildVarsToMap(slice []string) ([]schema.EnvKeyValueInput, error) {
+	result := []schema.EnvKeyValueInput{}
+
+	for _, entry := range slice {
+		// Split the entry by "="
+		parts := strings.Split(entry, "=")
```

Comment:
If the split is greater than 2, then we should concatenate everything after the first split